src/pkgcheck/checks/codingstyle.py

```diff
@@ -1035,3 +1035,42 @@ class EclassUnquotedVariablesCheck(_UnquotedVariablesCheck):
     def feed(self, eclass):
         for var_name, lines in self._feed(eclass):
             yield EclassUnquotedVariable(var_name, lines=lines, eclass=eclass.name)
+
+
+class ExcessiveLineLength(results.LinesResult, results.Style):
+    """Line is longer than 80 characters."""
+
+    line_length = 120
+    word_length = 110
+
+    @property
+    def desc(self):
+        return f'excessive line length over {self.line_length} {self.lines_str}'
+
+
+class ExcessiveLineCheck(Check):
+    """Scan files for lines that are longer than 80 characters."""
```

Comment:
```suggestion
    """Scan files for lines that are longer than 120 characters."""
```

Reply:
We should maybe avoid duplicating the '120' across ExcessiveLineLength and ExcessiveLineCheck.

Maybe we should rename the line to "LineLengthCheck"? I'm not sure what you think about checks correlating to result names, it might be that we want to keep that property...